doc/conf.py

```diff
@@ -259,7 +260,10 @@
 #html_search_scorer = 'scorer.js'
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'zephyrdoc'
```

Comment:
we don't use the HTML help or manpage builders at all as far as I know; why not just remove them?

Reply:
There definitely are no build targets for these. But since these options are not commented out and have "zephyr" in them, it would be better to also modify them to be consistent with the logic/thinking of this PR (which is simply to add the ability to generate kernel documentation only).